dia/build.rs

```diff
@@ -0,0 +1,8 @@
+fn main() {
+    windows::core::build!({
+        Microsoft::Dia::*,
+        Windows::Win32::Foundation::*,
+        Windows::Win32::System::Com::*,
+        Windows::Win32::System::SystemServices::*,
```

Comment:
```suggestion
```

Remove these - they're ignored - I should add a compiler error for this...

Reply:
Thanks, missed cleaning that up. Grr.